hw/ip/rom_ctrl/dv/cov/rom_ctrl_cov_excl.el

```diff
@@ -18,3 +18,11 @@ Transition Checking->RomAhead "341->917"
 ANNOTATION: "UNR"
 Fsm state_d "1410404563"
 Transition KmacAhead->ReadingHigh "629->181"
+
+// In rom_ctrl_scrambled_rom, the wire scr_nonce is having a fixed value and is a part of RHS of 
```

Comment:
We chatted earlier, but there are a couple of lines with trailing whitespace which cause the CI failure.